libraries/helpers.rb

```diff
@@ -0,0 +1,129 @@
+module OslRT
+  module Cookbook
+    module Helpers
+      # Take in a hashmap containing the properties we'd like to set
+      # for the RT instance, and convert to the perl config standard
+      # Returns a string
+      def parse_config(hOptions)
+        strConfig = ''
+        hOptions.each do |key, value|
+          # If the key is _Plugins, this is a special case
+          if key == '_Plugins'
+            # _Plugins contains an array of all plugins requested
+            strConfig += parse_plugin(value)
+            next
+          end
+          # If the key is _Lifecycles, this is a special case
+          if key == '_Lifecycles'
+            # _Lifecycles contains a recursive key-val/array pair for defining
+            # the progress of a ticket
+            strConfig += parse_lifecycle(value)
+            next
+          end
+          strConfig += "Set(#{key}, "
+          # Check to see if the key's first character is asking for a literal.
+          if key[0] == '%'
+            strConfig += "#{value});\n"
+            next
+          end
+          # Normal config option, check to see if it should be incapsulated with quotation marks.
+          strConfig += if !value.is_a?(String)
+                         # Interprete as a literal
+                         "#{value});\n"
+                       else
+                         # Add quotation marks
+                         "'#{value}');\n"
+                       end
+        end
+        strConfig
+      end
+
+      # Take in an array of plugins to add
+      # and return a string to append to the config file
+      def parse_plugin(arrPlugins)
+        strConfig = ''
+        arrPlugins.each do |plugin|
```

Comment:
Unless this causes an issue, sorting is usually good.
```suggestion
        arrPlugins.sort.each do |plugin|
```